Documentation/ApiOverview/SiteHandling/SiteSettingDefinitions.rst

```diff
@@ -8,88 +8,187 @@ Site settings definitions
 =========================
 
 ..  versionadded:: 13.1
+    Site-scoped setting definitions where introduced. They will most likely be
+    the place to configure site-wide configuration, which was previously only
+    possible to modify via Constant Editor, modifying TypoScript constants.
 
-The big problem with TypoScript and TSConfig is that each specified value can
-only ever be a string. It is up to the developer alone to read these values
-and convert them into the desired data type such as integer.
+Site settings definitions allow to define settings with a type and a guaranteed
+default value. They can be defined in :ref:`site-sets`, in a file called
+:file:`_blog_settins.definitions.yaml`.
```

Comment:
The file is wrong. It's `settings` and without `blog` and without `_` in front.

Reply:
Good catch!